### ISSUE TITLE
t-rex 0.25 beta ETH ALPH dual

### ISSUE THREAD

**Issue by Maythecoffee** [trexminer/T-Rex#1030]
Hello,
 nice thing, it is  working for woolly polly  and ,maybe for some other pools, but how to connect to my node, normally i use 
alphstratum+tcp://myiphere:10973.
how can i apply it in new t-rex?

**Comment by dedizones** [comment]
T-Rex v0.25.0 BETA (added blake3algorithm for mining Alephium coin; added ETH+ALPH dual mining mode for LHR cards ratio by default ETH 68% / ALPH 32%; NOTES: 1) only Alephium mining pools supported in this build are Woolypooly and Herominers; 2) This version for testing purposes only and need to be selected manually in miner configuration)

**Comment by alen84204** [comment]
> 

maybe u can try Woolypooly single mining

**Comment by dedizones** [comment]
i try Alephium and Ho my fu**ing god, this crypto is very sensitive to overclocking then I don't understand why in 2022 choosing such an energy-intensive algorithm, we are at a time when crypto POWs are frowned upon and we are releasing energy-intensive tokens..! I don't know if trex will be optimized at this level but we are on similar consumption to FIRO...

**Comment by Maythecoffee** [comment]
> T-Rex v0.25.0 BETA (added blake3algorithm for mining Alephium coin; added ETH+ALPH dual mining mode for LHR cards ratio by default ETH 68% / ALPH 32%; NOTES: 1) **only Alephium mining pools supported in this build are Woolypooly and Herominers**; 2) This version for testing purposes only and need to be selected manually in miner configuration)

i see a difference between , "this release supported 2 pools. " and "this release is working only with 2 pools and does not support full nodes!"

ALPH and TON both simple in OC,  just set memory -2000, core +200 (for some cars +250), ans set PL 115 for 3060, 125-140 for 3060ti, 220-240 for 3080. the simplest OC setting i saw.

**Comment by dedizones** [comment]
> > T-Rex v0.25.0 BETA (added blake3algorithm for mining Alephium coin; added ETH+ALPH dual mining mode for LHR cards ratio by default ETH 68% / ALPH 32%; NOTES: 1) **only Alephium mining pools supported in this build are Woolypooly and Herominers**; 2) This version for testing purposes only and need to be selected manually in miner configuration)
> 
> i see a difference between , "this release supported 2 pools. " and "this release is working only with 2 pools and does not support full nodes!"
> 
> ALPH and TON both simple in OC, just set memory -2000, core +200 (for some cars +250), ans set PL 115 for 3060, 125-140 for 3060ti, 220-240 for 3080. the simplest OC setting i saw.

yes I have the same settings. but here I find that it heats up a lot and consumes a lot.

I do ergo in multi and solo now suddenly apart from ETH the others if we are looking for maximum profitability, heat and consumption are there

**Comment by trexminer** [comment]
@Maythecoffee solo mining to nodes isn't supported

**Comment by Maythecoffee** [comment]
> @Maythecoffee solo mining to nodes isn't supported

Thanks for reply, any plans to add this feature?

**Comment by claudiodelf** [comment]
Guys a question about ETH + ALPH dual mining mode for LHR cards ratio by default ETH 68% / ALPH 32%.

Can you change the default to ETH 72% and ALPH 28% from additional miner's setting?

**Comment by trexminer** [comment]
> > @Maythecoffee solo mining to nodes isn't supported
> 
> Thanks for reply, any plans to add this feature?

Not planned for the near future, but I'll leave the issue open as a feature request

**Comment by alen84204** [comment]
> Guys a question about ETH + ALPH dual mining mode for LHR cards ratio by default ETH 68% / ALPH 32%.
> 
> 
> 
> Can you change the default to ETH 72% and ALPH 28% from additional miner's setting?

Cool idea, seems like such a good profitability

**Comment by mgrom** [comment]
Hi there, any news? When I was solo mining I've got like 20-30% more profit than through pools. It would be great to have it supported
